src/jit/lclvars.cpp

```diff
@@ -77,6 +77,9 @@ void                Compiler::lvaInit()
     lvaSIMDInitTempVarNum = BAD_VAR_NUM;
 #endif // FEATURE_SIMD
     lvaCurEpoch = 0;
+#ifdef FEATURE_MULTIREG_STRUCT_ARGS
```

Comment:
The idea for this and the previous was to have the caching var defined for MULTIREG case and the current usage for it to be for AMD64_SYSTEMV_STRUCT_PASSING only, but making it consistent it won't hurt. The ARM64 target could redefine the var for the MULTIREG case if it needs it.